schema/crdb/blueprint-add-sled-state/up2.sql

```diff
@@ -0,0 +1,13 @@
+set local disallow_full_table_scans = off;
+
+-- All sleds are considered active (this migration is in support of
+-- transitioning active-but-expunged sleds to 'decommissioned'). We'll fill in
```

Comment:
```suggestion
-- At this point in history, all sleds are considered active and this table only exists to support
-- transitioning active-but-expunged sleds to 'decommissioned'). We'll fill in
```

(easy to misread as "this [database] migration is in support of ...", in which case it's surprising for the new state to always be 'active')